matrix_client/api.py

```diff
@@ -774,6 +776,70 @@ def get_download_url(self, mxcurl):
         else:
             raise ValueError("MXC URL did not begin with 'mxc://'")
 
+    def media_download(self, mxcurl):
+        """ Download raw media from provided mxc URL
```

Comment:
You should get rid of the space after `"""`, and add a `.` at the end of this line (applies to all of your docstrings).